### DISCUSSION TITLE
umi4 qiankun useQiankunStateForSlave函数 导出给子应用的变量如何动态修改？

### DISCUSSION THREAD

**Discussion by fiorayang** [umijs/umi#11423] [Q&A]
主应用会给子应用传递一些配置信息，需要运行时修改，请问可以如何实现呢？

**Comment by fz6m** [comment]
参考文档使用 [`useQiankunStateForSlave`](https://umijs.org/docs/max/micro-frontend#%E4%B8%BB%E5%BA%94%E7%94%A8%E9%80%8F%E4%BC%A0%E6%95%B0%E6%8D%AE) 即可，当主应用 `setState` 的时候，子应用就可以在 `useModel('@@qiankunStateFromMaster').globalState` 拿到最新的数据。
